core/trino-spi/src/test/java/io/trino/spi/testing/TestInterfaceTestUtils.java

```diff
@@ -19,6 +19,7 @@
 
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+@SuppressWarnings("UnusedVariable")
```

Comment:
Don't suppress warnings. It hides potential legitimate problems and should only be used as a last resort.